rllib/core/models/catalog.py

```diff
@@ -25,93 +25,6 @@
 from ray.rllib.utils.spaces.space_utils import get_base_struct_from_space
 
 
-def _multi_action_dist_partial_helper(
```

Comment:
Moved this to end of file for cleanliness: Main classes should always "greet" the user first when they come to a file, not some obscure helper method.